rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -386,6 +384,9 @@ http {
 
     ssl_ecdh_curve {{ $cfg.SSLECDHCurve }};
 
+    ssl_certificate     /etc/ingress-controller/ssl/default-fake-certificate.pem;
+    ssl_certificate_key /etc/ingress-controller/ssl/default-fake-certificate.pem;
```

Comment:
Should this not be based on `servers["_"].SSLCert`?

Reply:
No. This contains the default fake certificate or the one configured with the flag --default-ssl-certificate. I will change the name of the file to avoid confusion.